src/puyapy-stubs/_deploy_time_variables.pyi

```diff
@@ -0,0 +1,7 @@
+from puyapy import Bytes, UInt64
+
+class UInt64DeployVar(UInt64):
```

Comment:
These could maybe just be functions rather than classes?

Reply:
Functions might be better, as we want to avoid non protocol inheritance of the stub types

Reply:
hmmmmm interesting, yeah I think a function might be the way to go here, if it all works the same with the expression builders (which it should I assume?)